README.md

```diff
@@ -30,3 +31,55 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+1. Recurrence Relation:
+
+   $T(n) = 1, \text{ if } n \leq 1$
+   
+   $T(n) = 3T\left(\frac{n}{3}\right) + n^5, \text{ when } n > 1$
+
+2. Expanding Recurrence Relation:
+
+   $T(n) = 3T\left(\frac{n}{3}\right) + n^5$
+
+   $= 3\left(3T\left(\frac{n}{9}\right) + \frac{n^5}{3}\right) + n^5$
```

Comment:
$\frac{n^5}{3}$ doesn't look right -- why is it $3$ and not $3^5$?

Reply:
Thank you it is $3^5$ now.